templates/title/show.json.twig

```diff
@@ -0,0 +1,46 @@
+{
+    "@context": "http://schema.org/",
+    "@type": "Book",
+    "@id": "{{ url('title_show', {'id':title.id}) }}",
+    "name": "{{title.title}}"
+    {% if title.pubdate %}, "datePublished": "{{title.pubdate}}" {% endif %},
+    "locationCreated": {
+        "@type": "Place"
+        {% if title.locationOfPrinting %}, "name": "{{title.locationOfPrinting}}" {% endif %},
```

Comment:
Same as in person/show.json.twig: the conditional should be moved up to encapsulate all of locationCreated.